engine/consensus/matching/engine.go

```diff
@@ -511,11 +567,6 @@ func (e *Engine) checkingSealing() error {
 		Str("next_unsealed_results", nextUnsealeds.String()).
 		Logger()
 
-	if len(sealableResults) == 0 {
-		lg.Info().Msg("checking sealing finished. No results can be sealed yet")
-		return nil
-	}
```

Comment:
This might be a fix to the potential sealing slowness. 

The reason to remove is that, if consensus nodes are behind, there isn’t any incoporated results in the mempool, then the requestPendingReceipt will never be triggered, therefore never be able to catch up.